frontend/src/screens/admin/workspaces/edit-study.tsx

```diff
@@ -37,20 +37,64 @@ const Boolean:FC<{val?: boolean}> = ({ val }) => {
 
 }
 
+type InfosProps = {
+    infos: AnalysisInfo[]
+    setFiles: SetFiles
+    stages: Stage[]
+}
+
+const Infos:FC<InfosProps> = ({ infos, setFiles, stages }) => {
+    const api = useApi()
+    const onDelete = (id: number) => {
+        api.adminDestroyInfo({ id }).then(resp => setFiles(resp))
+    }
+
+    if (!infos.length) return null
+
+    return (
+        <table className="table table-striped table-hover">
+            <thead>
+                <tr>
+                    <th scope="col">Stage #</th>
+                    <th scope="col">Uploaded</th>
+                    <th scope="col">Download</th>
+                    <th scope="col"></th>
+                </tr>
+            </thead>
+            <tbody>
+                {infos.map(info => (
+                    <tr key={info.id}>
+                        <th>{(stages.find(s => s.id == info.stageId)?.order || 0)+ 1}</th>
+                        <th scope="row">{formatDate(info.createdAt)}</th>
+                        <td>
+                            <a href={info.url} target="_blank"><Icon icon="cloudDownload" /></a>
```

Comment:
I actually don't know much about the `download` attribute as I haven't used it much - mainly just commenting on this for my own self learning if you have any more info about it beyond [what's in the docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#download). Maybe just control over the filename?

Reply:
the file will download due to the mime-type and content-disposition that's sent from the server

Reply:
however it probably wouldn't hurt to add `download`